.github/workflows/build-and-test.yml

```diff
@@ -242,6 +242,14 @@ jobs:
         source env/activate
         ttrt ${{ matrix.build.name }} ${{ matrix.build.ttrt_flags }} ${{ steps.strings.outputs.build-output-dir }}/test/ttmlir/Silicon --host-only
 
+    - name: Show Test Report
+      uses: mikepenz/action-junit-report@v4
+      if: success() || failure()
+      with:
+        report_paths: ttrt_report.xml
```

Comment:
Can you also upload ttrt report to artifacts?

Reply:
resolved on main

Reply:
done